.azure-pipelines/default.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+
+    displayName: 'Prep Environment, Install Packages copy'
+
+  - powershell: |
+     $inputFilterSet="$(build_targeting_string)"
+     
+     if ($inputFilterSet -eq "")
+     {
+         $inputFilterSet = "azure-*"
+     }
+     
+     foreach($inputFilter in "$inputFilterSet".split(","))
+     {
+       $foldersToIterate=Get-ChildItem -Recurse -Filter "$inputFilter"
+     
+       foreach ($folder in $foldersToIterate)
+       {
+         Write-Host python ./build_package.py --dest "$(Build.ArtifactStagingDirectory)" "$folder"
```

Comment:
I assume you are using Write-Host for debugging purposes as it will not execute that command.

Reply:
Apparently.